TrailBlaze/View Controllers/HomeViewController.m

```diff
@@ -225,7 +231,15 @@ - (NSString *) secondsToHMS: (int ) seconds {
     return [NSString stringWithFormat:@"%02d:%02d:%02d", seconds/3600, (seconds % 3600)/60, (seconds % 3600)%60];
 }
 
-
+- (void) onTimer {
+    if (isCurrentlyRunning) {
+        PFGeoPoint *userLocationGeoPoint = [[PFGeoPoint alloc] init];
```

Comment:
Can we initialize this with the lat and long instead of adding them on as properties?

`PFGeoPoint *const userLocationGeoPoint = [[PFGeoPoint alloc] initWithLatitude:locationManager.location.coordinate.latitude longitude:locationManager.location.coordinate.longitude];`

Reply:
The only init methods it has are init and initWithCoder, there was not init lat, long method.